### ISSUE TITLE
Add support for `jsr` registry and `jsr:` protocol for the imports. 

### ISSUE THREAD

**Issue by JayaKrishnaNamburu** [jspm/jspm#2619]
When installing some packages from `denoland`. Few of the packages are using `jsr:` registry and the protocol for its imports. This is an issue to keep track for the progress of `jsr` support in `@jspm/generator`

```js
import { Generator } from "@jspm/generator";

const generator = new Generator({
  defaultProvider: "denoland",
  defaultRegistry: "npm",
});
await generator.install("denoland:oak/body.ts");
console.log(generator.getMap());

```

[Sandbox](https://codesandbox.io/p/devbox/elated-saha-forked-3sywh7?file=%2Findex.js%3A1%2C1-9%2C1) to reproduce the same

Once the implementation is done, the tests need to be re-enabled in [@jspm/generator](https://github.com/jspm/generator/blob/main/test/providers/deno.skipbrowser.test.js#L3) and [jspm-cli](https://github.com/jspm/jspm-cli/blob/main/test/providers.test.ts#L48) respectively.

**Comment by guybedford** [comment]
Yes, these would be great to support! I have some prototype work, sounds like a good next step after we complete the release and update for the integrity story.

**Comment by guybedford** [comment]
WIP at https://github.com/jspm/generator/pull/372.

**Comment by jollytoad** [comment]
Are there any plans to add jsr packages into the cdn?

**Comment by guybedford** [comment]
This is not being actively worked on currnetly, but can be done if there is enough interest.
